reference-version/action.yaml

```
@@ -23,7 +23,7 @@ runs:
           echo "REF=${GITHUB_REF}" >> $GITHUB_ENV;
           echo "VERSION=${GITHUB_REF##refs/*/}" >> $GITHUB_ENV;
         fi;
-        echo "::warning title=reference-version is deprecated:: The reference-version action is deprecated. Please use ${{ github.ref_name }} instead. See https://docs.github.com/en/actions/learn-github-actions/contexts#github-context for more details."
+        echo "::warning title=reference-version is deprecated:: The reference-version action is deprecated. Please use `github.ref_name` instead. See https://docs.github.com/en/actions/learn-github-actions/contexts#github-context for more details."
```

```suggestion
        echo "::warning title=reference-version is deprecated:: The reference-version action is deprecated. Please use ${{ '${{ github.ref_name }}' }} instead. See https://docs.github.com/en/actions/learn-github-actions/contexts#github-context for more details."
```

Should do the trick, according to https://github.com/orgs/community/discussions/26621#discussioncomment-3252557